### PR TITLE
fix spelling of reverse proxy

### PR DESCRIPTION
**Pull request by DylanVerstraete**

### CODE REVIEW

File: JumpscaleLibs/clients/explorer/models/tfgrid_workloads_reservation_1.toml [hunk 1]
```diff
@@ -29,7 +29,7 @@ zdbs = (LO) !tfgrid.workloads.reservation.zdb.1
 networks = (LO) !tfgrid.workloads.reservation.network.1
 kubernetes = (LO) !tfgrid.workloads.reservation.k8s.1
 proxies = (LO) !tfgrid.workloads.reservation.gateway.proxy.1
-reserve_proxies = (LO) !tfgrid.workloads.reservation.gateway.reserve_proxy.1
+reserve_proxies = (LO) !tfgrid.workloads.reservation.gateway.reverse_proxy.1
```
Comment (zaibon) [inline JumpscaleLibs/clients/explorer/models/tfgrid_workloads_reservation_1.toml]:
```suggestion
reverse_proxies = (LO) !tfgrid.workloads.reservation.gateway.reverse_proxy.1
```

**Review by zaibon — CHANGES_REQUESTED** [review]

File: JumpscaleLibs/sal/reservation_chatflow/reservation_chatflow.py [hunk 2]
```diff
@@ -393,7 +393,7 @@ def is_finished(reservation):
             count += len(reservation.data_reservation.containers)
             count += len(reservation.data_reservation.kubernetes)
             count += len(reservation.data_reservation.proxies)
-            count += len(reservation.data_reservation.reserve_proxies)
+            count += len(reservation.data_reservation.reverse_proxies)
```
Comment (grimpy) [inline JumpscaleLibs/sal/reservation_chatflow/reservation_chatflow.py:422]: there's another one in this file on line 855

Comment (DylanVerstraete) [reply JumpscaleLibs/sal/reservation_chatflow/reservation_chatflow.py:422]: @grimpy  Line 855 does not exist?

Comment (grimpy) [reply JumpscaleLibs/sal/reservation_chatflow/reservation_chatflow.py:422]: https://github.com/threefoldtech/jumpscaleX_libs/blob/ded1bcc2af60e7ff226d642eb21568a49d339007/JumpscaleLibs/sal/reservation_chatflow/reservation_chatflow.py#L887

Comment (DylanVerstraete) [reply JumpscaleLibs/sal/reservation_chatflow/reservation_chatflow.py:422]: Wait, what? I can't see that... This code is not checked in on my branch: 
https://github.com/threefoldtech/jumpscaleX_libs/blob/development_fix_reverseproxy_spelling/JumpscaleLibs/sal/reservation_chatflow/reservation_chatflow.py

Comment (grimpy) [reply JumpscaleLibs/sal/reservation_chatflow/reservation_chatflow.py:422]: I assume you branched of from an old revision than, rebase and you will have it...

**Comment by DylanVerstraete** [comment]
@zaibon @grimpy  can we merge?

**Comment by grimpy** [comment]
@DylanVerstraete this should be merged at the same time as the zos changes get merged and deployed on dev/testnet

**Review by grimpy — APPROVED** [review]

**Review by zaibon — APPROVED** [review]

**Comment by zaibon** [comment]
dev/testnet updated

**Pull request merged**